configurator/src/main/java/com/devonfw/tools/ide/migrator/Migrations.java

```diff
@@ -111,7 +111,25 @@ public static MigrationImpl devon4j() {
             "implements $1Dao, io.oasp.module.jpa.common.base.LegacyDaoQuerySupport<$1Entity>",
             FileFilterPattern.reject("Application(MasterData)?DaoImpl\\.java")) //
         .and() //
-        .next().build();
+        .next().to(VersionIdentifier.ofDevon4j("3.1.0")) //
+        .pom().replaceProperty("devon4j.version", "3.1.0") //
+        .replaceProperty("spring.boot.version", "2.1.4.RELEASE") //
```

Comment:
according to https://github.com/devonfw/devon4j/issues/12 we are now on `2.1.6.RELEASE`.